packages/manager/apps/web/client/app/hosting/multisite/order/hosting-multisite-order.html

```diff
@@ -43,11 +43,13 @@
                     class="oui-radio__label-container"
                     for="hostingDomainAction-attach"
                 >
-                    <span class="oui-radio__icon"></span
-                    ><span
-                        class="oui-radio__label"
-                        data-translate="hosting_dashboard_add_or_order_step1_attach"
-                    ></span>
+                    <span class="oui-radio__label">
+                        <span class="oui-radio__icon"></span
```

Comment:
i think the fix needs to be applied elsewhere 
```
git grep -e 'class="oui-radio__label"' -c packages/manager/apps/web/
packages/manager/apps/web/client/app/domain-operation/update/domain-operation-update.html:3
packages/manager/apps/web/client/app/domain/redirection/add/domain-redirection-add.html:9
packages/manager/apps/web/client/app/domain/subdomain/add/domain-subdomain-add.html:3
packages/manager/apps/web/client/app/domain/zone/record/DKIM/domain-zone-record-DKIM.html:7
packages/manager/apps/web/client/app/domain/zone/record/DMARC/domain-zone-record-DMARC.html:2
packages/manager/apps/web/client/app/domain/zone/record/SPF/domain-zone-record-SPF.html:9
packages/manager/apps/web/client/app/domain/zone/reset/domain-zone-reset.html:2
packages/manager/apps/web/client/app/domains/dnssec/batch/domains-dnssec-batch.html:2
packages/manager/apps/web/client/app/email-domain/email/responder/create/email-domain-email-responder-create.html:4
packages/manager/apps/web/client/app/email-domain/email/responder/update/email-domain-email-responder-update.html:2
packages/manager/apps/web/client/app/email-domain/mailing-list/create/email-domain-mailing-list-create.html:6
packages/manager/apps/web/client/app/email-domain/mailing-list/update/email-domain-mailing-list-update.html:6
packages/manager/apps/web/client/app/hosting/bandwidth/order/hosting-bandwidth-order.html:1
packages/manager/apps/web/client/app/hosting/database/add/hosting-database-add.html:2
packages/manager/apps/web/client/app/hosting/database/import/hosting-database-import.html:2
packages/manager/apps/web/client/app/hosting/database/order/hosting-database-order.html:1
packages/manager/apps/web/client/app/hosting/ftp/user/update/hosting-ftp-user-update.html:3
packages/manager/apps/web/client/app/hosting/multisite/add/hosting-multisite-add.html:2
packages/manager/apps/web/client/app/hosting/multisite/order/hosting-multisite-order.html:2
packages/manager/apps/web/client/app/hosting/ovhconfig/hosting-ovhconfig.html:2
packages/manager/apps/web/client/app/hosting/ssl/order/hosting-order-ssl.html:3
packages/manager/apps/web/client/app/hosting/statistics/STATISTICS.html:1
packages/manager/apps/web/client/app/private-database/database/import/private-database-database-import.html:2
packages/manager/apps/web/client/app/private-database/database/ram/update/private-database-database-ram-update.html:1
packages/manager/apps/web/client/app/private-database/order/private-database-order.html:1
```

Maybe the CLI tool could be updated to take this case into account

Reply:
It was one left that @JeremyDec fixed with this commit.
All others cases has been covered on the "complex changes" commit. I've recheck all after he spotted this one.
I'll move this fix in this commit on the final rebase before the merge into `develop`